docs/source/release/v5.1.0/index.rst

```diff
@@ -64,7 +64,7 @@ Changes
 
   - :doc:`MantidPlot <mantidplot>`
 
-  - :doc:`Workbench <mantidworkbench>`
```

Comment:
Could we keep this as `mantidworkbench`? Although we internally say workbench we've used mantidworkbench in most places like the application name to avoid a name that is too generic like "workbench".